roles/generate_manifests/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+- name: Render nmstate
+  ansible.builtin.include_tasks: static.yml
+  loop: "{{ groups['nodes'] }}"
+
+- name: Make cluster-manifests dir
+  ansible.builtin.file:
+    name: "{{ manifests_dir }}"
```

Comment:
This task is causing problems in our CI lab where we execute the playbook from a central location but we have different bastion hosts for each cluster.  manifests_dir is generated 
```
roles/post_install/defaults/main.yml:manifests_dir: "{{ generated_dir }}/{{ cluster_name }}"
```
generated_dir is

```
roles/post_install/defaults/main.yml:generated_dir: "{{ repo_root_path }}/generated"
```

Our repo_root_path only exists on the host launching the ansible-playbook command.  So the above fails.

This doesn't happen in the regular assisted install code path which uses a similar path call fetched_dest
```
roles/create_cluster/defaults/main.yml:fetched_dest: "{{ repo_root_path }}/fetched"
```

The tasks that create the fetched_dest and use it all delegate_to: localhost.

I've been staring at the roles for a while now and I'm not sure how to tackle this.  I do believe we want these files to come back to where the playbook is being executed from.  Since we sometimes run additional playbooks after that interact with the cluster.

Thoughts?

Reply:
I believe delegating those tasks to localhost (e.g. https://github.com/redhat-partner-solutions/crucible/blob/1e4a21783d28c452e4105271457355540b1f9d76/roles/generate_ssh_key_pair/tasks/main.yml#L7-L13 )is the proper way to go since it is already being handled that way for onprem AI